js/react_native/ios/OnnxruntimeModule.mm

```diff
@@ -43,7 +50,33 @@ @implementation OnnxruntimeModule
     NSDictionary *resultMap = [self loadModel:modelPath options:options];
     resolve(resultMap);
   } @catch (...) {
-    reject(@"onnxruntime", @"can't load model", nil);
+    reject(@"onnxruntime", @"failed to load model", nil);
+  }
+}
+
+/**
+ * React native binding API to load a model using BASE64 encoded model data string.
+ *
+ * @param modelData the BASE64 encoded model data string
+ * @param options onnxruntime session options
+ * @param resolve callback for returning output back to react native js
+ * @param reject callback for returning an error back to react native js
+ * @note when run() is called, the same modelPath must be passed into the first parameter.
+ */
+RCT_EXPORT_METHOD(loadModelFromBase64EncodedBuffer
+                  : (NSString *)modelDataBase64EncodedString options
+                  : (NSDictionary *)options resolver
+                  : (RCTPromiseResolveBlock)resolve rejecter
+                  : (RCTPromiseRejectBlock)reject) {
+  @try {
+    // Choices: if not working, use 3rd party library for base64 encoding/decoding:
+    // https://github.com/nicklockwood/Base64/ (may be deprecated)
+    // https://github.com/l4u/NSData-Base64
```

Comment:
Do we need this info anymore?

Reply:
sorry overlooked old comments before. cleaned up. same before.